security/pkg/pki/ca/controller/secret.go

```diff
@@ -262,14 +270,15 @@ func (sc *SecretController) scrtUpdated(oldObj, newObj interface{}) {
 		return
 	}
 
-	ttl := time.Until(cert.NotAfter)
+	certLifeTimeLeft := time.Until(cert.NotAfter)
```

Comment:
For readability, can we just calculate the `gracePeriod` here, rather than having the calculation in the `if` condition below?

Reply:
Done.